app/views/shopping_list/index.html.erb

```diff
@@ -0,0 +1,12 @@
+<%= turbo_frame_tag "modal" do %>
+  <div class="modal">
+    <div>
+      <%= link_to "Close", :back, class: 'btn btn-outline-danger'%>
+    </div>
+    <h1>Generate Shopping List</h1>
+    <%= form_with(scope: "inventory") do |f| %>
+      <%= f.select :id, @inventories.map { |c| [c.name, c.id] } %>
+      <%= f.submit "Generate Shopping List" %>
```

Comment:
:red_circle: Kindly make sure When the user clicks on this button He should navigate to a new page (Shopping List page ) :arrow_down:
 
<img src="https://github.com/microverseinc/curriculum-rails/raw/main/recipe-app/images/inventory_shopping_list.png">